### ISSUE TITLE
Memory Issues with Panel app

### ISSUE THREAD

**Issue by vntkumar8** [holoviz/panel#1399]
Hi, 
I have built a panel app.  When I connect to my app via the browser, the app is executed and takes up some memory. When I close the tab, open another tab and connect again, the memory used increases. If I keep doing this, the memory consumption keeps increasing until the machine runs out of memory.
I have attached a diagnostic plot that shows memory usage, I connected via one browser, closed it then commented few others then closed them also. But memory doesn't return to baseline. 
![image](https://user-images.githubusercontent.com/7812689/84384129-21359980-ac0b-11ea-92ff-bd829b68ba63.png)

Any workarounds for this memory consumption issue?

**Comment by philippjfr** [comment]
I'd have to see more details about your app. Sessions should definitely be cleaned up after you close the browser, but it may take a little (like a minute or two). Based on that plot that doesn't seem to be happening in your case.

**Comment by vntkumar8** [comment]
A Small example
```
#!/usr/bin/env python
# coding: utf-8

# In[57]:


import pandas as pd
import numpy as np
import panel as pn
import hvplot.pandas

import holoviews as hv

import requests
import requests_cache
import gc
import csv

import colorcet as cc
from colorcet.plotting import swatch

from pandas_cache import pd_cache

pn.extension()

gc.enable()


# In[58]:


df = pd.DataFrame([81,9,2,12,213,2312,123,12,13213,13213,1767,5665,345])


# In[59]:


barplot = df.hvplot(kind='bar', responsive=True, height=300).opts(color='Green',
                                                            xlabel='Week', ylabel='%',xrotation=45,
                                                            axiswise=True, toolbar='above',
                                                            default_tools=['save','zoom_out','reset'] 
                                                            )


# In[60]:


dash = pn.Tabs(pn.Pane(barplot))


# In[61]:


dash.servable()


# In[ ]:




```

panel serve example.py --num-procs 1  --log-level debug --mem-log-frequency 15000

You will find that documents and models remain uncollected.

**Comment by philippjfr** [comment]
Thank you!

**Comment by philippjfr** [comment]
#1407 should fix these issues. I'm now debating whether to backport part of that fix and release 0.9.6 or wait for 0.10.0.

**Comment by jbednar** [comment]
In the meantime, can you tag a dev release so that we can evaluate this fix on our deployments?
